.circleci/config.yml

```diff
@@ -12,9 +12,18 @@ workflows:
           npm-run: lint
       - node/test:
           matrix:
+            alias: Run Tests
             parameters:
               version:
                 - "12.22"
                 - "14.19"
                 - "16.13"
                 - "lts"
+          requires: ["Validate Build", "Lint"]
+      - node/run:
```

Comment:
@stephenh , it looks like there isn't a CircleCI context set up for the `stephenh` organization in CircleCI. Could you either:

a) create one for NPM publishing, or
b) manually add `GITHUB_TOKEN` and `NPM_TOKEN` to this single project

The docs on what those tokens need to be are described here: https://semantic-release.gitbook.io/semantic-release/usage/ci-configuration#authentication

Reply:
Done, hopefully